man/homectl.xml

```diff
@@ -506,6 +506,40 @@
         <xi:include href="version-info.xml" xpointer="v247"/></listitem>
       </varlistentry>
 
+    <varlistentry>
+        <term><option>--blob-directory=</option><replaceable>PATH</replaceable></term>
+        <term><option>-b</option> <replaceable>PATH</replaceable></term>
+        <term><option>--blob-directory=</option><replaceable>FILENAME</replaceable>=<replaceable>PATH</replaceable></term>
+        <term><option>-b</option> <replaceable>FILENAME</replaceable>=<replaceable>PATH</replaceable></term>
+
+        <listitem><para>Accepts either a directory path, or a file name followed by a file path. If just a
+        directory path is specified, then the user's entire blob directory is replaced the specified path.
+        Note that this replacement is performed before per-file manipulations are applied, which means these per-file
+        manipulations will be applied on top of the specified directory. If a filename and file path are specified, then
+        the single specified file in the blob directory will be overwritten with the specified path. If completely
+        blank, all changes to the blob directory up to this point are reset. If a filename is specified but the corresponding
```

Comment:
> If completely blank, all changes to the blob directory up to this point are reset

Does this behavior make sense?

`homectl update user --blob-directory=/path/to/newblob --avatar=/path/to/avatar --blob-directory=foobar=/path/to/foobar --blob-directory=` should be a no-op

Alternatively `--blob-directory=` can mean "delete the existing contents of the blob directory". Could be used along w/ the file-specific forms to blank out the blob directory and then apply files over top

Let me know which behavior you prefer